deployment/update_production.sh

```diff
@@ -43,7 +43,7 @@ sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py dumpdata --natural-foreign
 
 sudo -H -u "$USERNAME" "$ENVDIR/bin/pip" install -r requirements.txt
 # sometimes, this fails for some random i18n test translation files.
-sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py compilemessages || true
+sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py compilemessages
 sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py scss --production
 sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py ts compile --fresh
```

Comment:
There's still the discussion about the `|| true`s here, right?

@Kakadus did I understand you correctly that some steps failed here in the past, but the "Test update" workflow still passed?
This feels weird to me: This has `set -e` to abort on error. So, the test workflow should have a step with a non-zero return code, where it should stop and fail. Why did this not happen?

Reply:
Yes. when you look at [this backup process run](https://github.com/e-valuation/EvaP/actions/runs/3934713350/jobs/6729702916#step:8:59), the `scss` and `ts complie` command actually "fail", but still the return code must be zero. The previous version use a python container whereas now I use the setup-python action. This is probably due to some github action internal. However, I will try to investigate